.github/workflows/reusable-workflow-terraform.yml

```
@@ -352,3 +350,25 @@ jobs:
         shell: bash
         run: |
           terraform apply -input=false -auto-approve -no-color
+
+      - name: Extract PR number from commit message
+        if: failure() && steps.terraform_apply.outcome == 'failure'
+        id: extract_pr
+        run: |
+          PR_NUMBER=$(git log -1 --pretty=%B | grep -oP 'Merge pull request #\K\d+')
```

```suggestion
          PR_NUMBER=$(git log -1 --pretty=%B | grep -oP '#\K\d+')
```
Our default pull request message is "pull request title and commit details", if you look at the commits to main over time (https://github.com/ministryofjustice/analytical-platform/commits/main/), they do not conform to "Merge pull request #<PR>"

```bash
$ PAGER="" git log -1 --pretty=%B                   
:dependabot: terraform(deps): Bump hashicorp/aws from 5.55.0 to 5.56.1 in /terraform/auth0/dev-analytics-moj (#4612)


$ git log -1 --pretty=%B | grep -oP '#\K\d+'        
4612
```